app/livechat/server/statistics/LivechatAgentActivityMonitor.js

```diff
@@ -0,0 +1,86 @@
+import moment from 'moment';
+import { Meteor } from 'meteor/meteor';
+
+import { callbacks } from '../../../callbacks/server';
+import { LivechatAgentActivity, Sessions, Users } from '../../../models/server';
+
+const formatDate = (dateTime = new Date()) => ({
+	date: parseInt(moment(dateTime).format('YYYYMMDD')),
+});
+
+export class LivechatAgentActivityMonitor {
+	constructor() {
+		this._handleMeteorConnection = this._handleMeteorConnection.bind(this);
+		this._handleAgentStatusChanged = this._handleAgentStatusChanged.bind(this);
+		this._handleUserStatusLivechatChanged = this._handleUserStatusLivechatChanged.bind(this);
+	}
+
+	start() {
+		this._setupListeners();
```

Comment:
Would not it be good to have a property to store when the `monitor` starts? Otherwise, in case the `start()` is called again, new callbacks will be assigned and duplicated, right?